Source/WebCore/platform/graphics/FloatSize.h

```diff
@@ -60,7 +61,7 @@ class FloatSize {
 public:
     FloatSize() { }
     FloatSize(float width, float height) : m_width(width), m_height(height) { }
-    WEBCORE_EXPORT FloatSize(const IntSize&);
+    FloatSize(const IntSize& size) : m_width(size.width()), m_height(size.height()) { }
```

Comment:
You can either change this and the line above to have them aligned with the style guidelines, or leave as is.

Reply:
Once it run through EWS then I will change both in one go. :-)

Reply:
Fixed both.